packages/frontend/src/components/create-campaign/form/rewards-step/index.tsx

```diff
@@ -66,6 +67,13 @@ export function RewardsStep({
         rewardToken?.address,
     );
 
+    const totalRewardsUsdAmount = useMemo(() => {
+        if (!rewards) return 0;
+        return rewards.reduce((accumulator, reward) => {
+            return (accumulator += reward.amount * reward.price);
```

Comment:
What happens here if `price` is null?

Reply:
Fixed in a06f031f998ba0c7d995b892b4e399b596601e9d.